src/content/docs/logs/log-api/introduction-log-api.mdx

```diff
@@ -475,6 +475,7 @@ Restrictions on logs sent to the Log API:
 Some specific attributes have additional restrictions:
 
 * `accountId`: This is a reserved attribute name. If it is included, it will be dropped during ingest.
+* `appId`: Must be an integer. If you set a string or other type, the data is ingested but becomes unqueryable.
```

Comment:
What is "other type" here? I think that could do with just a touch of clarification, but other than that looks good!

Reply:
Ops, other data type hehe, added a commit, thanks @jeff-colucci!